apps/publish_mdm/admin.py

```diff
@@ -27,9 +28,10 @@
 
 @admin.register(CentralServer)
 class CentralServerAdmin(admin.ModelAdmin):
-    list_display = ("base_url", "created_at", "modified_at", "organization")
+    list_display = ("base_url", "created_at", "modified_at", "organization", "username")
```

Comment:
Perhaps we should leave out the username (or display a truncated version of it), since it's encrypted in the database?